ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PORT=8080
+
+exec java -jar -Dserver.port="${PORT}" "refund-request-consumer.jar"
```

Comment:
Doesn't this need to be

```suggestion
exec java -jar -Dserver.port="${PORT}" "refund-request-consumer-java.jar"
```

Reply:
I know in the Makefile it's set to `refund-request-consumer.jar` as the artifact name but I'm just wondering if we should keep it all the same as the repo name and what the task will be called in ECS

Reply:
Yeah that's a good point. I've updated the name so it should now be `refund-request-consumer-java` everywhere